datacenter/ucp/3.0/guides/admin/install/install-on-azure.md

```diff
@@ -0,0 +1,286 @@
+VM that's part of the Kubernetes cluster to be configured with a pool of
+IP addresses.
+
+You have two options for deploying the VMs for the Kubernetes cluster on Azure:
+- Install the cluster on Azure stand-alone virtual machines. Docker UCP provides
+  an [automated mechanism](#configure-ip-pools-for-azure-stand-alone-vms)
+  to configure and maintain IP pools for stand-alone Azure VMs. 
+- Install the cluster on an Azure virtual machine scale set. Configure the
+  IP pools by using an ARM template like 
+  [this one](#set-up-ip-configurations-on-an-azure-virtual-machine-scale-set).
+
+The steps for setting up IP address management are different in the two
+environments. If you're using a scale set, you set up `ipConfigurations`
+in an ARM template. If you're using stand-alone VMs, you set up IP pools
+for each VM by using a utility container that's configured to run as a
+global Swarm service, which Docker provides.
+
+## Azure Prerequisites 
+
+The following list of infrastructure prerequisites need to be met in order 
+to successfully deploy Docker UCP on Azure.
+
+- All UCP Nodes (Managers and Workers) need to be deployed into the same 
+Azure Resource Group. The Azure Networking (Vnets, Subnets, Security Groups) 
+components could be deployed in a second Azure Resource Group.
+- All UCP Nodes (Managers and Workers) need to be attached to the same 
+Azure Subnet.
+- All UCP (Managers and Workers) need to be tagged in Azure with the 
+`Orchestrator` tag. Note the value for this tag is the Kubernetes version number
+in the format `Orchestrator=Kubernetes:x.y.z`. This value may change in each 
+UCP release. To find the relevant version please see the UCP 
+[Release Notes](../../release-notes). For example for UCP 3.0.6 the tag 
+would be `Orchestrator=Kubernetes:1.8.15`. 
+- The Azure Computer Name needs to match the Node Operating System's Hostname. 
+Note this applies to the FQDN of the host including domain names. 
+- An Azure Service Principal with `Contributor` access to the Azure Resource 
+Group hosting the UCP Nodes. Note, if using a separate networking Resource 
+Group the same Service Principal will need `Network Contributor` access to this 
+Resource Group.
+
+The following information will be required for the installation:
+
+- `subscriptionId` - The Azure Subscription ID in which the UCP 
+objects are being deployed. 
+- `tenantId` - The Azure Active Directory Tenant ID in which the UCP 
+objects are being deployed. 
+- `aadClientId` - The Azure Service Principal ID
+- `aadClientSecret` - The Azure Service Principal Secret Key
+
+### Azure Configuration File
+
+For Docker UCP to integrate in to Microsoft Azure, an Azure configuration file 
+will need to be placed within each UCP node in your cluster. This file 
+will need to be placed at `/etc/kubernetes/azure.json`. 
+
+See the template below. Note entries that do not contain `****` should not be 
+changed.
+
+```
+{
+    "cloud":"AzurePublicCloud", 
```

Comment:
Should we note that `AzurePublicCloud` works for public Azure regions, but would need to be adjusted for sovereign clouds (US Gov, Germany, China, etc.)? 

Not sure if it's a 1:1 mapping as the first entry doesn't quite match, but in the Azure CLI the results of `az cloud list` are `AzureCloud`, `AzureChinaCloud`, `AzureUSGovernment`, `AzureGermanCloud`

Reply:
@stevenfollis do you have specific customers using the sovereign clouds? I would say that someone has to spin it up and try these cloud labels before we can document it, but we should not hold publishing these docs to get this specific item in.

Reply:
Agreed, tossing out the potentials for further improvements in the future. 

The biggest "sovereign cloud" is the US Government cloud, used extensively by Public Sector clients. 

We have some multi-national customers that are either already or looking to deploy in the Azure China Cloud (Tiffany deploying today, Amway planning to post-PoC). Would also suspect some of the EMEA folks would know some customers in the German datacenter.

Reply:
@mark-church and @stevenfollis: if this is resolved, please mark it as such. Thanks.